client/src/components/MenuBar/components/DiffExpButtons/constants.ts

```diff
@@ -0,0 +1,6 @@
+export const DIFFEXP_SLOW_MSG =
+  "(CAUTION: large dataset - may take longer or fail)";
+export const genTipMessage = (slowMsg: string) =>
+  `See top differentially expressed genes${slowMsg}`;
+export const genTimMessageWarn = (cellCountMax: number) =>
```

Comment:
Should this be `genTipMessageWarn`?

Reply:
Oops... yes it should - thanks for spotting this 🙏